src/App.js

```diff
@@ -1,25 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import { Calculator } from './components/Calculator';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code>src/App.js</code>
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="container">
+      <Calculator />
```

Comment:
- [ ] Please make the app.js a class based component

Reply:
Hi @AshakaE, thank you for pointing this out 🙏 ! I have gone ahead and edited the code accordingly.